python/ray/serve/_private/client.py

```diff
@@ -415,10 +419,12 @@ def delete_all_apps(self, blocking: bool = True):
 
     @_ensure_connected
     def delete_deployments(self, names: Iterable[str], blocking: bool = True) -> None:
+        """Delete 1.x deployments."""
+
         ray.get(self._controller.delete_deployments.remote(names))
         if blocking:
             for name in names:
-                self._wait_for_deployment_deleted(name)
+                self._wait_for_deployment_deleted(name, "")
```

Comment:
Shouldn't this be default app name here?

Reply:
Since this method `client.delete_deployments` is only used for 1.x deployments, the application name should be `""`